recipes/dd-handler.rb

```diff
@@ -32,10 +32,7 @@
 chef_gem 'chef-handler-datadog' do # ~FC009
   action :install
   version node['datadog']['chef_handler_version']
-  # Chef 12 introduced `compile_time` - remove when Chef 11 is EOL.
-  compile_time true if respond_to?(:compile_time)
```

Comment:
As the comment indicated, this `compile_time` should not be removed yet. Chef 11 is still supported by Chef and has not reached EOL.